patches/aesthetics.py

```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
         "#####, how do I download RAM?",
         "Is this a delayed April Fool's Joke?",
         "Dayoman is close to getting 120% in 1:20!",
-        "Play as if your footage will go in a\nSummoning Salt video."
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",

```

Comment:
People not aware of what cheating means for insiders might get the wrong idea.

Reply:
Asking Matty about it now.

Reply:
I understand and agree.